src/test/java/io/lettuce/core/commands/ReadOnlyCommandIntegrationTests.java

```diff
@@ -0,0 +1,61 @@
+package io.lettuce.core.commands;
+
+import io.lettuce.core.TestSupport;
+import io.lettuce.core.api.sync.RedisCommands;
+import io.lettuce.core.cluster.ClusterReadOnlyCommands;
+import io.lettuce.core.cluster.ClusterTestUtil;
+import io.lettuce.core.cluster.api.StatefulRedisClusterConnection;
+import io.lettuce.core.cluster.api.sync.RedisClusterCommands;
+import io.lettuce.core.protocol.CommandType;
+import io.lettuce.core.protocol.ProtocolKeyword;
+import io.lettuce.core.protocol.ReadOnlyCommands;
+import io.lettuce.test.KeyValueStreamingAdapter;
+import io.lettuce.test.LettuceExtension;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.inject.Inject;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author Thach Le
+ */
+@ExtendWith(LettuceExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ReadOnlyCommandIntegrationTests extends TestSupport {
+
+    private final RedisCommands<String, String> redis;
+
+    @Inject
+    protected ReadOnlyCommandIntegrationTests(RedisCommands<String, String> redis) {
+        this.redis = redis;
+    }
+
+    @BeforeEach
+    void setUp() {
+        redis.flushall();
+    }
+
+    @Test
+    void testReadOnlyCommands() {
```

Comment:
Awesome! This is exactly what I wanted to see and it would help us keep this list up-to-date.

Two notes:
- you can move this test to the `ServerCommandIntegrationTest` - no need to create a whole new class for it
- you can use the `CommandDetailParser` to parse the result, instead of parsing it yourself (would also mean that if something changes in the result API you would not have to modify the test and we will keep the change locally to the parser)

Reply:
Hi @tishun, thanks for the notes.
I updated the test, please help check 🙇

Reply:
btw, If anything need to pass the CI check, please suggest.
On local, I got this message:
```
[ERROR] Failed to execute goal net.revelc.code.formatter:formatter-maven-plugin:2.16.0:validate (default) on project lettuce-core: File '/...lettuce/src/test/java/io/lettuce/core/commands/ServerCommandIntegrationTests.java' has not been previously formatted.  Please format file and commit before running validation! -> [Help 1]

```